File: organization-service/src/main/java/com/mealmap/organizationservice/core/dto/organization/OrganizationUpdatingDto.java
package com.mealmap.organizationservice.core.dto.organization;

import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
public class OrganizationUpdatingDto {
    private Integer upn;

    private String name;

    private String legalAddress;

    private String phoneNumber;

    private String email;
}

File: frontend/src/api/organization/dto/OrganizationUpdatingDto.ts
export interface OrganizationUpdatingDto {
    upn: number; // Must be positive and not null
    name: string; // Between 2 and 100 characters
    legalAddress: string; // Between 2 and 150 characters
    phoneNumber: string; // Between 3 and 20 characters
    email: string; // Valid email format
}